enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/context/HeartbeatContextImpl.java

```diff
@@ -141,6 +141,13 @@ public void notify( HeartbeatListener listener )
     @Override
     public void suspicions( InstanceId from, Set<InstanceId> suspicions )
     {
+        // Ignoring this message if it is the first received from a failed instance. It will obviously suspect everyone.
+        if ( isFailed( from ) )
+        {
+            getInternalLog( HeartbeatContext.class ).info( "Ignoring suspicions from failed instance " + from );
```

Comment:
I'm not sure failed instance will ignore everyone. This heartbeat timeout dependent. Let's just log suspicions as well.